cluster/gce/gci/configure-helper.sh

```diff
@@ -2856,7 +2864,14 @@ function main() {
     create-node-pki
     create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME}
     if [[ "${KUBE_PROXY_DAEMONSET:-}" != "true" ]]; then
-      create-kubeproxy-user-kubeconfig
+      if [[ -n "${KUBE_PROXY_TOKEN:-}" ]]; then
+        create-kubeproxy-user-kubeconfig
+      elif [[ -f "/var/lib/kubelet/kubeconfig" ]]; then
```

Comment:
In the default case, CREATE_BOOTSTRAP_KUBECONFIG will be true and /var/lib/kubelet/kubeconfig will not exist at this point.

Are we actually exercising this case (no KUBE_PROXY_TOKEN, but also no boostrap kubeconfig)? This seems to be adding additional config paths that will not actually be used.